dbapi/src/main/java/com/griddynamics/jagger/dbapi/dto/DecisionPerTestGroupDto.java

```diff
@@ -0,0 +1,52 @@
+package com.griddynamics.jagger.dbapi.dto;
+
+import com.griddynamics.jagger.dbapi.entity.DecisionPerTaskEntity;
+import com.griddynamics.jagger.dbapi.entity.TaskData;
+import com.griddynamics.jagger.util.Decision;
+
+import java.util.List;
+
+public class DecisionPerTestGroupDto {
```

Comment:
"DecisionPerTestGroupDto should extend DecisionPerTaskDto" - no, it should't. It's not a data model, it's dto. Its structure must describe JSON structure. If extend DecisionPerTaskDto then all fields will be copied to DecisionPerTestGroupDto and then there won't be any difference between them.

Removing DecisionPerTestGroupDto is also bad. It'll lead to some bad consequences:
1. We won't be able to distinguish test group from test
2. As far as all entities will have children list of the same entities, it'll be easier to make mistake filling them since there's no difference between parent and children.

I think, test group and test must have their own dtos - it'll be clear and simple.